src/ui/styles/_header.scss

```diff
@@ -56,18 +56,22 @@ header {
   }
 }
 
+main.os-darwin:not(.native-frame) .col-main-controls {
+  padding-left: 65px; // let some space for titleBarStyle
+}
 
 header {
 
   .col-main-controls {
-    width: 200px;
+    width: 220px;
     display: flex;
     align-items: center;
+    padding-right: 10px;
   }
 
   /* .windows-controls */
   .col-player-infos {
-    width: calc(100% - 400px);
+    width: calc(100% - 440px);
```

Comment:
Slightly off-topic, but I think we might eventually want to use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/var)

Reply:
Totally, I planned to refactor these SCSS files with variables and other best pratices

Reply:
Ideally I'd prefer CSS variables since they are first class in the browser where SCSS vars only exist in the code